src/anet.c

```diff
@@ -588,11 +588,15 @@ int anetUnixAccept(char *err, int s) {
     return fd;
 }
 
-int anetPeerToString(int fd, char *ip, size_t ip_len, int *port) {
+int anetFdToString(int fd, char *ip, size_t ip_len, int *port, int fd_to_str_type) {
```

Comment:
While at it, the safety of this function should probably improved by using `snprintf()` instead of `strncpy()` and making sure we always end up with a null terminator.

Reply:
done